RockWeb/Webhooks/Mailgun.ashx

```diff
@@ -78,19 +78,26 @@ public class Mailgun : IHttpHandler
             
             if ( !string.IsNullOrWhiteSpace( request.Form["workflow_action_guid"]))
             {
-                Guid? actionGuid = request.Form["workflow_action_guid"].AsGuidOrNull();
+                Guid? actionGuid = request.Form["workflow_action_guid"].Split( ',' )[0].AsGuidOrNull();
                 string status = string.Empty;
```

Comment:
@jreciomoreno , Not sure why there would be more than one guid here, this is set by the SendEmailWithEvents workflow action, and it looks like it would only ever have one guid.